config/routes.rb

```diff
@@ -45,6 +45,11 @@
       get 'urs_search'
     end
   end
+
+  namespace :api, constraints: { format: 'json' }  do
+    resource :collections, only: [:create]
+  end
```

Comment:
Is creating this route specifically for UMF to post draft collections? Does it solve the authenticity token issue?

Reply:
https://github.com/MAAP-Project/mmt/blob/42a223f5195759f7975513e3a97c796578411737/app/controllers/api/base_controller.rb#L1 <-- This line I think is the important thing to avoid the authenticity token verification.

The base_controller in the API is not inheriting from the ActionController:Base but the API controller which doesn't include CSRF by default protection, and from what I understand it doesn't make sense that an API would implement CSRF